### ISSUE TITLE
Autodetect different rubocop binary for different projects

### ISSUE THREAD

**Issue by phlegx** [AtomLinter/linter-rubocop#283]
I'm using RVM (Ruby version manager) to manage my ruby installations and and to manage the bundler gem sets. It would be handy that `linter-rubcop` could detect the proper `Rubocop` binary depending on the project I have opened in `Atom`. Is there a way that linter-rubocop package can handle that?

**Comment by danmayer** [comment]
yeah this has become a big issue for using atom rubocop as I have a bunch of different projects that are all on different ruby versions. So when I have open two at once there is a high likelyhood that one is showing rubocop errors.

**Comment by vzamanillo** [comment]
It is a good proposal, that involves

- the execution of `rvm-exec current` to know which is the current gemset (ex: `2.3.1`)
- the execution of Rubocop in the current gemset  with `rvm 2.3.1 do rubocop`

does anyone know another or better approach?

**Comment by blowfishpro** [comment]
I was actually able to get this to work by just changing Atom's config for `linter-rubocop`.  This works for RVM, not sure if other ruby version managers have an equivalent
```
"*":
  "linter-rubocop":
    command: "rvm in . do rubocop"
```

The key point here is that `linter-rubocop` executes with the right working directory, but that doesn't have the same effect as `cd`ing there in a shell.

**Comment by vzamanillo** [comment]
@blowfishpro, Thank you for your suggestion, Joseph!! very useful.

**Comment by 73** [comment]
> I was actually able to get this to work by just changing Atom's config for `linter-rubocop`. This works for RVM, not sure if other ruby version managers have an equivalent
> 
> ```
> "*":
>   "linter-rubocop":
>     command: "rvm in . do rubocop"
> ```
> 
> The key point here is that `linter-rubocop` executes with the right working directory, but that doesn't have the same effect as `cd`ing there in a shell.

This should be in the documentation.
